plugins/woocommerce/client/legacy/css/woocommerce-blocktheme.scss

```diff
@@ -0,0 +1,475 @@
+			}
+
+			#coupon_code {
+				width: 50%;
+				margin-right: 0;
+			}
+		}
+	}
+
+	.cart-collaterals h2 {
+		// Ensure cart subheadline size is reasonable.
+		font-size: var(--wp--preset--font-size--medium);
+	}
+
+	/**
+	* Checkout specific
+	*/
+	.woocommerce-form-coupon .button {
+		min-height: 50px;
+	}
+
+	/**
+	* Order confirmation
+	*/
+	.woocommerce-thankyou-order-received,
+	h2.woocommerce-column__title {
+		font-size: var(--wp--preset--font-size--large);
+		font-weight: 300;
+	}
+
+	ul.woocommerce-order-overview {
+		// Display order overview items next to each other.
+		display: flex;
+		width: 100%;
+		padding-left: 0;
+		font-size: var(--wp--preset--font-size--small);
+
+		@media only screen and ( max-width: 768px ) {
+			flex-direction: column;
+		}
+
+		li {
+			flex-grow: 1;
+			margin-bottom: 1rem;
+			border: none;
+		}
+	}
+
+	.woocommerce-customer-details address {
+		// Ensure customer details match order overview.
+		box-sizing: border-box;
+		width: 70%;
+		padding: 2rem;
+		border-width: 1px;
+		border-radius: 0;
+	}
+}
+
+/**
+* My account
+*/
+.woocommerce-account {
+	.woocommerce-MyAccount-navigation {
+		ul {
+			// Ensure top left alignment of the navigation.
+			margin: 0 0 2rem;
+			padding: 0;
+		}
+
+		li {
+			// Remove default list styling.
+			list-style: none;
+			padding: 1rem 0;
+
+			@media only screen and ( max-width: 768px ) {
+				padding: 0.35rem 0;
+			}
+
+			&:first-child {
+				padding-top: 0;
+			}
+
+			&.is-active {
+				a {
+					text-decoration: underline;
+				}
+			}
+
+			a {
+				text-decoration: none;
+
+				&:hover {
+					text-decoration: underline;
+				}
+			}
+		}
+	}
+
+	.woocommerce-MyAccount-content {
+		> p:first-of-type,
+		p.form-row-first,
+		p.form-row-last {
+			// Ensure first paragraph in content is top aligned.
+			margin-block-start: 0;
+		}
+	}
+
+	table.shop_table.order_details,
+	.woocommerce-customer-details address {
+		// Ensure order/customer details are full width in My Account.
+		width: 100%;
+	}
+
+	.addresses .title .edit {
+		// Align "Add" and "Edit" buttons when title is too long (/my-account/edit-address/).
+		display: block;
+		margin-bottom: 1rem;
+	}
+
+	&.woocommerce-edit-address .woocommerce-MyAccount-content form > h3 {
+		// Ensure headline is top aligned (/my-account/edit-address/shipping/).
+		margin-block-start: 0;
+	}
+}
+
+/**
+* My account - Login form
+*/
+.woocommerce-page {
+	.woocommerce-form-login {
+		.input-text {
+			// Ensure inputs are well spaced.
+			font-size: var(--wp--preset--font-size--small);
+			padding: 0.9rem 1.1rem;
+		}
+
+		label {
+			margin-bottom: 0.7rem;
+		}
+
+		.show-password-input {
+			// Adjust password field icon position.
+			top: 0.8rem;
+			right: 1.2rem;
+		}
+	}
+}
+
+/**
+* Store notice
+*/
+p.demo_store,
+.woocommerce-store-notice {
+	// Add theme colors to the store notice.
+	color: var(--wp--preset--color--background, $primarytext);
+	background: var(--wp--preset--color--foreground, $primary);
```

Comment:
> So, generally the current styling of the Store Notice bar is problematic out-of-the-box. While I considered reporting this in WooCommerce project ... I think that we could potentially be forward compatible here.

Interesting idea, but I guess that the best thing is creating another issue for discussing this problem. With this change, we could break some UX experiences.

Reply:
> Interesting idea, but I guess that the best thing is creating another issue for discussing this problem. With this change, we could break some UX experiences.

@gigitux The more I think about it, the more I think we should address it here. Since, we're looking to create a stylesheet that makes block themes integrate nicely with WooCommerce. 

Both [StoreFront](https://github.com/woocommerce/storefront/blob/trunk/assets/css/woocommerce/woocommerce.scss#L1728) _and_ [Twenty Twenty-Two](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/client/legacy/css/twenty-twenty-two.scss#L1255) use `position: fixed;`. 

Block theme are relatively new, and I'm not sure what UX we would be breaking? 🤔

Reply:
With ce5cd547134f03ee79f5dd98faa0525832c2238d I introduced your suggestion for fix notice position.